libminifi/include/utils/file/FileUtils.h

```diff
@@ -112,31 +112,33 @@ class FileUtils {
 #endif
   }
 
-  static std::string create_temp_directory(char *format) {
+  static std::string create_temp_directory(const char * const format) {
```

Comment:
This breaks backwards compatibility on POSIX because the buffer pointed-to by the argument is no longer modified. It was never modified on Windows, so the new behavior is consistent but different.
Is this OK to change?

Reply:
As it's test code and I think nobody has ever made anything that depended on the modified buffer, I'm fine with this.

Reply:
I bailed on the idea since too many processor tests rely on the buffer being modified and they continue to compile but fail at runtime.